plugins/symbol-processor/src/test/kotlin/org/jetbrains/dataframe/ksp/DataFrameSymbolProcessorTest.kt

```diff
@@ -888,6 +1043,173 @@ class DataFrameSymbolProcessorTest {
         }
     }
 
+    private val petstoreYaml = File("../../core/src/test/resources/petstore.yaml")
+    private val petstoreYamlUrl = "https://raw.githubusercontent.com/OAI/OpenAPI-Specification/main/examples/v3.0/petstore.yaml"
```

Comment:
is there a reason why this is downloaded directly from their github instead of using local copy + useHostedFile?

Reply:
I wanted to test whether that would work too, but local might be better